odata-data-aggregation-ext/3.3 Transformations Producing a Subset.md

```diff
@@ -260,7 +260,7 @@ results in
 
 The `top` transformation takes a non-negative integer $c$ as argument. Let $A$ be a copy of the input set with a total order that extends any existing order of the input set but is otherwise chosen by the service. The total order MUST be stable across requests.
 
-If $A$ contains more than $c$ instances, the output set consists of the first $c$ instances of $A$. Otherwise, the output set equals $A$. The instances in the output set are in the same order as they occur in $A$.
+If $A$ contains more than $c$ instances, the output set consists of the first $c$ instances in $A$ (possibly with repetitions). Otherwise, the output set equals $A$. The instances in the output set are in the same order as they occur in $A$.
```

Comment:
It is about occurrences, not instances. Therefore apply "occurrence" like at other places, e.g compute

Proposal: "the output set consists of the first $c$ occurrences in $A$ ."